src/web/src/index.tsx

```diff
@@ -1,8 +1,19 @@
 import { createRoot } from 'react-dom/client';
+import { HashRouter } from 'react-router';
 import App from './components/App.tsx';
 import './index.css';
 
 // Render the application. Currently we do not provide
 // runtime and ims details to the application, so it can't run deployed actions
 const root = createRoot(document.getElementById('root')!);
-root.render(<App />);
+
+/* **here you can mock the exc runtime and ims objects** */
```

Comment:
The app-builder-samples repository has an `index.js` file that tries to load the runtime within an Experience Cloud shell, and then falls back to a mock version if that fails. Should we be doing something similar?

Reply:
That's next on my list. I really don't like the way they've done it :laughing: